.github/workflows/test_wprocket_legacy.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       fail-fast: true
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.2']
-        wp-versions: ['5.7']
+        php-versions: ['7.3']
```

Comment:
This file can be removed now, and the WP version change should be done in the workflow which has the 7.3 version